src/main/resources/public/template/behaviours/sniplet-notes-memento-widget.html

```diff
@@ -0,0 +1,158 @@
+                <div ng-click="vm.previousPeriod()">
+                    <p class="sniplet arrow-nav">‹</p>
+                </div>
+                <div>
+                    <p ng-show="vm.isAverageSubjectsDisplay()" class="sniplet nav-period-name">
+                        [[vm.getSelectedPeriodLabel()]]</p>
+                    <p ng-show="vm.isSubjectDetailDisplay()"
+                       class="sniplet nav-matiere-name-subject-detail">
+                        [[vm.getSelectedMatiereName()]]</p>
+                    <p ng-show="vm.isSubjectDetailDisplay()"
+                       class="sniplet nav-period-name-subject-detail">[[vm.getSelectedPeriodLabel()]]</p>
+                </div>
+                <div ng-click="vm.nextPeriod()">
+                    <p class="sniplet arrow-nav">›</p>
+                </div>
+            </div>
+        </div>
+        <div class="sniplet content-notes">
+            <div ng-show="vm.isLastNotesDisplay()">
+                <div class="sniplet black-shadow" ng-show="vm.hasNote()">
+                    <div ng-repeat="note in vm.notes" class="sniplet display-flex content-last-note">
+                        <div ng-mouseenter="showCompetencesInfo=true" ng-mouseleave="showCompetencesInfo=false"
+                             ng-show="showCompetencesInfo" class="sniplet competences-info">
+                            <div>
+                                <div ng-repeat="competence in note.competences" class="sniplet competences-text">
+                                    <p>[[competence.nom]]</p>
+                                    <div ng-style="{'background': vm.getCompetencesColor(competence.evaluation)}"
+                                         class="sniplet competences-chip"></div>
+                                </div>
+                            </div>
+                        </div>
+                        <div class="sniplet left-content">
+                            <div class="sniplet info-note-content">
+                                <p class="font-bold">[[note.matiere]]</p>
+                                <p>[[note.teacher]]</p>
+                                <p>[[note.title]]</p>
+                                <p>[[vm.formatDate(note.date, "DD/MM/YYYY")]]</p>
+                            </div>
+                            <div ng-mouseenter="showCompetencesInfo=true" ng-mouseleave="showCompetencesInfo=false"
+                                 ng-show="vm.isNoteHasCompetences(note)" class="sniplet note-competences-indicator">
+                                <p>C</p>
+                            </div>
+                        </div>
+                        <div ng-style="{'background': vm.getNoteBackgroundColor(note)}"
+                             class="sniplet note-content">
+                            <p><span class="sniplet note-value font-bold">[[note.note]]</span>/[[note.diviseur]]</p>
```

Comment:
![Uploading note.PNG…]()

Reply:
C'est normal une évaluation peut-être Non Notée